sonar-php-plugin/build.gradle.kts

```diff
@@ -45,6 +45,7 @@ tasks.jar {
         "Plugin-Name" to "PHP Code Quality and Security",
         "Plugin-Organization" to "SonarSource",
         "Plugin-OrganizationUrl" to "https://www.sonarsource.com",
+        "Plugin-RequiredForLanguages" to "php",
```

Comment:
Found it in [this doc](https://docs.sonarsource.com/sonarqube/latest/extension-guide/developing-a-plugin/plugin-basics/#advanced-build-properties), looks like we could add it no? Looked at the Java and Python analyzer, they are both using it 🤔

Reply:
Yes, it's good to add it here